src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -775,4 +792,65 @@ public boolean isMiddlePrefooterOnPage() {
   public void verifyMiddlePrefooterAdPresent() {
     verifyAdVisibleInSlot(MIDDLE_PREFOOTER_CSS_SELECTOR, middlePrefooter);
   }
+
+  public void verifyUnlockCSS() {
+    String unlockCssUrl = "";
+    // Stylesheets with main scss package in href
+    String mainCssSelector = "link[href*=\"" + MAIN_CSS_PACKAGE + "\"]";
+    // Sibling to above selector which is link
+    String unlockCssSelector = mainCssSelector + " + link";
+
+    try {
+      driver.findElement(By.cssSelector(mainCssSelector));
+    } catch (Exception exception) {
+      PageObjectLogging.log("UnlockCSS", "Main css file not found", false, driver);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "UnlockCSS" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)